### ISSUE TITLE
[Enhancement] Add optional flags which can be used to turn on/off certain scans

### ISSUE THREAD

**Issue by emtunc** [emtunc/SlackPirate#6]
Add optional --flags that can be used to turn on/off certain scans/functions.
For example, if I only want to scan for AWS keys then I should be able to execute something like this:

`./SlackPirate.py --token 123 --aws-keys`

or if I want to scan for AWS keys and private keys:

`./SlackPirate.py --token 123 --aws-keys --private-keys`

**Comment by influenza** [comment]
I prefer the opposite approach. With only the `--token` parameter provided, all scans will run. Scans could be selectively disabled with `--no-files` or `--no-urls` for instances. This is more in alignment with other tools I've seen. It seems the common case would be to perform all scans. What do you think?

**Comment by emtunc** [comment]
Yup I am in agreement with that - I didn't explain it very well in the initial comment. Run everything by default but let the user select if they don't want to run a certain scan using --no-xyz

**Comment by influenza** [comment]
@emtunc would you be so kind as to create a feature branch for `option-flags` - we can pull in the work from my form and collaborate with @MartinIngesen and/or @milangfx to get the desired behavior in place

**Comment by emtunc** [comment]
@influenza done, I think.
